core/server/api/canary/users.js

```diff
@@ -156,6 +156,9 @@ module.exports = {
                 frame.options.transacting = t;
 
                 return models.Post.destroyByAuthor(frame.options)
+                    .then(() => {
+                        return models.ApiKey.destroyPersonalTokenByUser(frame.options);
```

Comment:
You should be able to use the [`destroy` method from the base model](https://github.com/TryGhost/Ghost/blob/master/core/server/models/base/index.js#L1074-L1089) here by passing a `destroyBy` object in the options! :relaxed: 
```js
return models.ApiKey.destroy({
  ...frame.options,
  destroyBy: {
    user_id: options.id
  }
});
```